learnEcl/0200-syntax.md

```diff
@@ -77,21 +77,21 @@ SomeResult;"
 
 ## Definition
 
-Assigning an expression to an attribute. Definitions can't not be executing unless it is wrapped in an action and are defined by `:=`. Let's take a look at an example:
+Assigning an expression to an attribute. Definitions cannot be executing unless it is wrapped in an action and are defined by `:=`. Let's take a look at an example:
 
-`Val := 23;` is a Definition. Attribute Val is defined and value 23 is assigned to it. To turn `Val` to an action we can wrap it in an OUTPUT.`OUTPUT(Val);` is an Action and result would be 23.
+`Val := 23;` is a Definition. Attribute `Val` is defined and value `23` is assigned to it. To turn `Val` to an action we can wrap it in an OUTPUT. `OUTPUT(Val);` is an Action and the result would be `23`.
```

Comment:
What is this "to turn Val into an action" thing?  Plus, OUTPUT does not have a return value; it creates an output in the workunit.  No actions have a return value; that is, in fact, what makes them 'actions'.

This part needs some re-thinking.